libs/dh/shared/ui-util/src/lib/dh-dropdown-translator.directive.ts

```diff
@@ -16,31 +16,23 @@
  */
 import { DestroyRef, Directive, Input, OnInit, inject } from '@angular/core';
 import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
-import { WattDropdownComponent } from '@energinet-datahub/watt/dropdown';
 import { TranslocoService } from '@ngneat/transloco';
-import { withLatestFrom } from 'rxjs';
+
+import { WattDropdownComponent } from '@energinet-datahub/watt/dropdown';
 
 @Directive({
   standalone: true,
   selector: '[dhDropdownTranslator]',
 })
 export class DhDropdownTranslatorDirective implements OnInit {
-  private trans = inject(TranslocoService);
+  private translocoService = inject(TranslocoService);
   private host = inject(WattDropdownComponent);
+  private destroyRef = inject(DestroyRef);
+
   @Input({ required: true }) translate = '';
-  destroyRef = inject(DestroyRef);
 
   ngOnInit(): void {
-    this.host.filteredOptions$
-      .pipe(
-        takeUntilDestroyed(this.destroyRef),
-        withLatestFrom(this.trans.selectTranslateObject<object>(this.translate))
-      )
-      .subscribe(([, keys]) => {
-        this.setTranslation(keys);
-      });
-
-    this.trans
+    this.translocoService
```

Comment:
Suggestion: I think this could be moved to the constructor. Then we wont not need the destroyRef

Reply:
Unfortunately the value of `translate` Input will not be set at this point.